contracts/platform/registry/Registry.sol

```diff
@@ -190,18 +196,19 @@ contract Registry is Destructible, RegistryInterface, Upgradable {
     }
 
     /// @dev get the endpoints of a provider
-    function getProviderEndpoints(address provider) public view returns (bytes32[]) {
+    function getProviderEndpoints(address provider) public view returns (bytes32[] memory) {
         return db.getBytesArray(keccak256(abi.encodePacked("oracles", provider, "endpoints")));
     }
 
     /// @dev get all endpoint params
-    function getEndpointParams(address provider, bytes32 endpoint) public view returns (bytes32[]) {
+    function getEndpointParams(address provider, bytes32 endpoint) public view returns (bytes32[] memory) {
         return db.getBytesArray(keccak256(abi.encodePacked('oracles', provider, 'endpointParams', endpoint)));
     }
 
     /// @dev get broker address for endpoint
     function getEndpointBroker(address oracleAddress, bytes32 endpoint) public view returns (address) {
-        return address(db.getBytes32(keccak256(abi.encodePacked('oracles', oracleAddress, endpoint, 'broker'))));
+         db.getBytes32(keccak256(abi.encodePacked('oracles', oracleAddress, endpoint, 'broker')));
```

Comment:
for some reason the original return with casting of db.getBytes32 on the return statement did not work.

had first let db.getBytes32 process then cast the return.  by returning address(db)